podpac/datalib/drought_monitor.py

```diff
@@ -0,0 +1,36 @@
+
+from podpac.core.data.types import Zarr
+from podpac.core.coordinates import ArrayCoordinates1d
+
+class DroughtMonitorCategory(Zarr):
```

Comment:
@mpu-creare Actually, one more note.

Reall, the DroughtMonitorCategory should implement its own get_time method in order to specify that the time coordinates are ctype 'left' and provide the segment lengths:

```
    def get_time(self):
        values = super(DroughtMonitorCategor, self).get_time()
        segment_lengths = ['%d,d' for val in values[1:]]
        segment_lengths.append('30,d') # for december
        return ArrayCoordinates1d(values, ctype='left', segment_lengths=segment_lengths)
```

We could even partially fix the precision issue here, something more like:

```
    def get_time(self):
        values = super(DroughtMonitorCategor, self).get_time()
        values = [convert_to_month_only(value) for value in values]
        return ArrayCoordinates1d(values, ctype='left', segment_lengths='1,M')
```

In theory, we want to get the June value for any datetime in June (rather than getting the nearest, which would be the July value for the second half of June if using ctype='midpoint' or ctype='point'), and this would be the way to indicate that. I'm not sure how the time interpolation is handling 'nearest' and if it is handling the ctypes correctly, though.

Reply:
just an observation:

@jmilloy i like the architecture using the `get_[dim]` in addition to the keys. We should push the `get_[dim]` to other similar DataSource types to allow the datalib to do some more customization of the key-val/tabular formats (CSV, Dataframe, hdf5)

Reply:
We should probably stage the data better for now...

Reply:
Do you want me to spend an hour restaging the data without using the cf conventions? That would've probably been easier that handling the cf decoding in the first place.